Upgrade Spanner samples to OpenCensus 0.25.0

## In which file did you encounter the issue?
spanner/cloud-client/pom.xml

### Did you change the file? If so, how?
Upgrade OpenCensus to v0.25.0 by RenovateBot in #2224 

## Describe the issue

There are two issues here, both w/ the Spanner Samples:
SpannerSampleIT
```
java.lang.IllegalAccessError: tried to access field io.opencensus.trace.unsafe.ContextUtils.CONTEXT_SPAN_KEY from class io.grpc.internal.CensusTracingModule$TracingClientInterceptor
	at com.example.spanner.SpannerSampleIT.setUp(SpannerSampleIT.java:61)
```
QuickStartSampleIT
```
com.google.cloud.spanner.SpannerException: UNKNOWN: tried to access field io.opencensus.trace.unsafe.ContextUtils.CONTEXT_SPAN_KEY from class io.grpc.internal.CensusTracingModule$TracingClientInterceptor
	at com.example.spanner.QuickstartSampleIT.testQuickstart(QuickstartSampleIT.java:58)
Caused by: com.google.cloud.spanner.SpannerException: UNKNOWN: tried to access field io.opencensus.trace.unsafe.ContextUtils.CONTEXT_SPAN_KEY from class io.grpc.internal.CensusTracingModule$TracingClientInterceptor
Caused by: java.lang.IllegalAccessError: tried to access field io.opencensus.trace.unsafe.ContextUtils.CONTEXT_SPAN_KEY from class io.grpc.internal.CensusTracingModule$TracingClientInterceptor
```

--- a/spanner/cloud-client/pom.xml
+++ b/spanner/cloud-client/pom.xml
@@ -33,7 +33,7 @@ limitations under the License.
     <maven.compiler.target>1.8</maven.compiler.target>
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <opencensus.version>0.23.0</opencensus.version>
+    <opencensus.version>0.25.0</opencensus.version>
   </properties>
 
   <!-- Include the BOM to ensure compatibility across multiple google-cloud-* libraries -->
@@ -62,6 +62,14 @@ limitations under the License.
         <exclusion>
           <groupId>io.opencensus</groupId>
           <artifactId>opencensus-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-netty-shaded</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-core</artifactId>
         </exclusion>
       </exclusions>
     </dependency>

